_rules/SC1-1-1-image-has-name.md

```diff
@@ -51,83 +51,99 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed example 1
 
-HTML `img` element has accessible name
+The HTML `img` element has an accessible name
 
 ```html
 <img alt="W3C logo" />
 ```
 
 #### Passed example 2
 
-Element with role of `img` and accessible name
+The element with role of `img` has an accessible name
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed example 3
 
-Accessible name but not always accessibility supported
+The element has an accessible name, though the name is not always accessibility supported
 
 ```html
 <img title="W3C logo" />
 ```
 
 #### Passed example 4
 
-HTML `img` element marked as [decorative](#decorative) through empty `alt` attribute
+The HTML `img` element is marked as [decorative](#decorative) through an empty `alt` attribute
 
 ```html
 <img alt="" />
 ```
 
 #### Passed example 5
 
-HTML `img` element marked as [decorative](#decorative) through `role="presentation"`
+The HTML `img` element is marked as [decorative](#decorative) through `role="presentation"`
 
 ```html
 <img role="presentation" />
 ```
 
 #### Passed example 6
 
-HTML `img` element marked as [decorative](#decorative) through `role="none"`
+The HTML `img` element is marked as [decorative](#decorative) through `role="none"`
 
 ```html
 <img role="none" />
 ```
 
+#### Passed example 6
+
+The HTML `img` element has an accessible name that does not only consist of whitespace
+
+```html
+<img alt=":-)" />
```

Comment:
I'm curious about this. I can think of situations where this could be a suitable alt attribute, though perhaps not the best alt attribute value. And correct punctuation should be used in alt attributes. The only aspect of SC1.1.1 this might not meet is H86, which is a technique rather than a success criteria (for my understanding).

That said, I do think a better example should be used.

Reply:
Please do keep in mind that this rule tests whether there is an accessible name or not.
It does not test whether the accessible name is descriptive. This should be tested by another rule.

I do think we need to document "edge cases" where we might disagree on which rule should be catching a particular WCAG violation. 
Or else we will be back in the mud where one tool/method says a rule is passed, while another says it is not applicable.

Reply:
@WilcoFiers, @kasperisager and @annethyme agreed on a call to write up a proposal that states that Passed rules should also be passing the accessibility requirements (and then we will have to ensure better coverage through Failed test cases instead). 
However, we do currently have many rules where this is not the case, and this will have to be corrected across the board for all rules. 

I suggest that when a proposal on this has been created and accepted, we clean up all rules. 
However, I would like to keep the rules in this PR as they are for now. 
I could go and clean up the newly added test cases to align them with this future proposal, but in many rules there will be existing test cases that also break this proposed requirement, and it will be messy if I clean it up halfway by doing it just for the newly added test cases, but not the old ones, and I will never finish work on this PR if I try to do it for all test cases for all rules in this PR.

Would @WilcoFiers be okay with me merging as is (since this is the only substantial thing that has come in during the Final Call), knowing that new guidelines for test cases might be on their way?